lib/stat_tracker.rb

```diff
@@ -33,9 +33,27 @@ def total_goals_per_game
     end
   end
 
+  def total_away_wins
+    away_wins = 0
+    @games.each do |game|
+      if game.away_goals > game.home_goals
```

Comment:
@hopesgit Once we have game_teams data pulled into StatTracker, you could also use that collection instead to mirror the calculation you've done in the spreadsheet (count of games that are  'away' and 'win'). Just a thought for the future!

Reply:
That's true! Might make it easier once game_teams is in stat tracker!

Reply:
Maybe? Does it account for ties? Even at this point, we can't just compare wins and losses because of ONE tie in the data.

Reply:
The method to which I'm referring is in my latest PR in case you're interested. It does account for ties (just filters them out). But this method you have here works well. I'm okay with keeping it as is. We can always look at refactoring mine or yours later for them to match. Yours may be better because it's iterating through a smaller collection.